doc/multiple_robot_arms/multiple_robot_arms_tutorial.rst

```diff
@@ -0,0 +1,606 @@
+        <include file="$(find gazebo_ros)/launch/empty_world.launch">
+            <arg name="use_sim_time" value="true" />
+            <arg name="gui" value="true" />
+            <arg name="paused" value="false" />
+            <arg name="debug" value="false" />
+        </include>
+
+        <!-- Find my robot Description-->
+        <param name="robot_description" command="$(find xacro)/xacro  '$(find panda_multiple_arms)/robot_description/panda_multiple_arms.xacro'" />
+
+        <!-- Spawn The robot over the robot_description param-->
+        <node name="urdf_spawner" pkg="gazebo_ros" type="spawn_model" respawn="false" output="screen" args="-urdf -param robot_description -model panda_multiple_arms" />
+        
+    </launch>
+
+2. Preparing a ``bringup_moveit.launch`` file to start the three integration components. Create the file in the ``panda_multiple_arms/launch`` directory. ::
+
+    cd ~/ws_moveit
+    cd src/panda_multiple_arms/launch 
+    touch bringup_moveit.launch
+
+Copy the following XML code to the ``bringup_moveit.launch`` file. ::
+    
+    <?xml version="1.0"?>
+    <launch>
+
+        <!-- Run the main MoveIt executable with trajectory execution -->
+        <include file="$(find panda_multiple_arms_moveit_config)/launch/move_group.launch">
+            <arg name="allow_trajectory_execution" value="true" />
+            <arg name="moveit_controller_manager" value="ros_control" />
+            <arg name="fake_execution_type" value="interpolate" />
+            <arg name="info" value="true" />
+            <arg name="debug" value="false" />
+            <arg name="pipeline" value="ompl" />
+            <arg name="load_robot_description" value="true" />
+        </include>
+
+        <!-- Start the simulated robot in an empty Gazebo world -->
+        <include file="$(find panda_multiple_arms)/launch/panda_multiple_arms_empty_world.launch" />
+
+        <!-- Start the controllers and robot state publisher-->
+        <include file="$(find panda_multiple_arms)/launch/control_utils.launch"/>
+
+        <!-- Start moveit_rviz with the motion planning plugin -->
+        <include file="$(find panda_multiple_arms_moveit_config)/launch/moveit_rviz.launch">
+            <arg name="rviz_config" value="$(find panda_multiple_arms_moveit_config)/launch/moveit.rviz" />
+        </include>
+
+    </launch>
+
+
+To start the MoveIt-Gazebo integration, build and source your ROS workspace, and run the ``bringup_moveit.launch`` file. ::
+
+    cd ~/ws_moveit
+    catkin build 
+    source devel/setup.bash
+    roslaunch panda_multiple_arms bringup_moveit.launch
+
+If all steps are done, this should bring up all the required components for the integration. Then, we can plan motions for the arms and hands using MoveIt's rviz plugin and execute those motions on the simulated robots in Gazebo as shown in `this video <https://www.youtube.com/watch?v=h8zlsuzeW3U>`_.
+
+
+Step 5: Plan arms motions with MoveIt Move Group Interface.
```

Comment:
Maybe you should omit this section because other tutorials already cover it.

Or you can shorten the section quite a bit, like this:

```
When writing code for multiple move groups, motion planning works the same way it did in previous tutorials. There is an `example here for multiple move groups.<https://github.com/Robotawi/panda_arms_ws/blob/master/src/panda_multiple_arms/src/plan_simple_motion.cpp>`_
```

Reply:
I agree with you the section was redundant. It is now shortened.

Reply:
I want to add this line before the last line in the tutorial to the motion planning section.
 
```
This example uses a separate move group for every arm, but we can make a new move group that contains both the ``right_arm`` and ``left_arm`` groups to plan and execute motions for them simultaneously.

```

What do you think? @AndyZe

Reply:
That sounds great. I can do it now through the web interface to save you some time.